test/api/resources/accounts.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import APIResource from 'api/apiResource';
+import AccountResource from 'api/resources/accounts';
+
+describe('AccountResouce', () => {
+	const defaultBasePath = 'http://localhost:1234';
+	const defaultPath = '/accounts';
+
+	let LiskAPI;
+	let resource;
+
+	beforeEach(() => {
+		LiskAPI = {
+			headers: {},
+			fullURL: defaultBasePath,
+			hasAvailableNodes: () => {},
+			randomizeNodes: () => {},
+			banActiveNodeAndSelect: () => {},
+		};
+		resource = new AccountResource(LiskAPI);
+	});
+
+	describe('#constructor', () => {
+		it('should throw error without LiskAPI input', () => {
```

Comment:
What if it's called with something, but it's not an instance of `LiskAPI`?

Reply:
That would be a problem.
However, importing lisk-api from resource will have circular dependency.
Any suggestions?